lib/Config.php

```diff
@@ -282,12 +280,9 @@ public static function getWebsiteConfig($language = null)
                 }
 
                 //TODO resolve for all langs, current lang first, then no lang
-                $config = new PimcoreConfig($settingsArray, true);
-
-                Cache::save($config, $cacheKey, $cacheTags, null, 998);
-            } elseif ($config instanceof PimcoreConfig) {
-                $data = $config->toArray();
-                foreach ($data as $key => $setting) {
+                Cache::save($settingsArray, $cacheKey, $cacheTags, null, 998);
```

Comment:
![image](https://user-images.githubusercontent.com/6014195/187520489-72454fc6-0cc6-4a05-81b3-8a66f0285070.png)
Isn't here missing a `$config = $settingsArray;` (replacing `$config = new PimcoreConfig($settingsArray, true);`) otherwise the config at 295 `self::setWebsiteConfig($config, $language);` is crashing due being a `bool` and the `return $config` at line 298 would be returning `bool`

Reply:
@robertSt7 @mcop1

Reply:
I've opened https://github.com/pimcore/pimcore/pull/13032 to better manage and see this potential issue (and leaving alone the merged pr/archived task 😄 )

Reply:
Thanks for creating the pr, I assigned it to myself and I will have a look right away.